regression/cbmc-viewer/test.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+"""
+Regession testing of CBMC-viewer.
```

Comment:
We have added some text: The script is guarding against a simple textual diff of the html that would fail if cbmc were to make some reasonable change that simply changed the text of the trace produced.  The script is parsing the html and comparing only the shapes of the result.